sploosh-wasm/src/lib.rs

```diff
@@ -181,13 +178,10 @@ impl PossibleBoards {
         prior_steps_from_previous_stddevs: &[f64],
     ) -> Vec<f64> {
         // We must compute the boards and their corresponding probabilities from our observations.
-        let mut board_priors: Vec<f64> = Vec::with_capacity(604584);
-        for _ in 0..604584 {
-            board_priors.push(0.0);
-        }
+        let mut board_priors = vec![0.0; 604584];
```

Comment:
if you want this to be efficient, you should probably just be using a stack allocating array e.g.
```rust
let mut board_priors = [0.0f32; 604584];
```

and below